sdk/ios/push-notifications.md

```diff
@@ -176,110 +243,46 @@ Make the following changes to the configuration file. The values are available i
-            @escaping (UIBackgroundFetchResult) -> Void) {
+2. Go to your Membership details page to get the Team ID
 
-            pinpoint!.notificationManager.interceptDidReceiveRemoteNotification(
-                userInfo, fetchCompletionHandler: completionHandler)
+3. Run `amplify notifications add`
 
-            if (application.applicationState == .active) {
-                let alert = UIAlertController(title: "Notification Received",
-                                            message: userInfo.description,
-                                            preferredStyle: .alert)
-                alert.addAction(UIAlertAction(title: "Ok", style: .default, handler: nil))
-
-                UIApplication.shared.keyWindow?.rootViewController?.present(
-                    alert, animated: true, completion:nil)
-            }
-        }
-
-        // Request user to grant permissions for the app to use notifications
-        func registerForPushNotifications() {
-            UNUserNotificationCenter.current().delegate = self
-            UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound, .badge]) {
-                (granted, error) in
-                print("Permission granted: \(granted)")
-                // 1. Check if permission granted
-                guard granted else { return }
-                // 2. Attempt registration for remote notifications on the main thread
-                DispatchQueue.main.async {
-                    UIApplication.shared.registerForRemoteNotifications()
-                }
-            }
-        }
-
-        // Other app delegate methods...
-    }
-    ```
-
-    **Note**: If you already have push notification delegate methods, you can just add the `interceptDidRegisterForRemoteNotifications` and `interceptDidReceiveRemoteNotification` callbacks to Pinpoint client.
-
-1. In Xcode Project Navigator, choose your app name at the top, choose your app name under **Targets**, choose the **Capabilities** tab, and then turn on **Push Notifications**.
-
-    ![Image of turning on Push Notifications capabilities in Xcode]({{image_base}}/xcode-turn-on-push-notification.png)
-
-1. Configure the app to run in the **Release** profile instead of the default **Debug** profile. Perform the following steps to get a notification to the device:
-
-    1. For your app target, go to the **General** tab of project configuration and make sure **Automatically Manage Signing** check box is not selected.
-
-    1. In the **Signing (Release)** section, choose the production provisioning profile you created on Apple developer console. For testing push notifications on a device, you will need an [Ad Hoc Provisioning Profile](https://help.apple.com/xcode/mac/current/#/dev4335bfd3d) configured with a Production AppStore and Ad Hoc certificate, and with the device(s) to be used for testing.
+```
+? Choose the push notification channel to enable. `APNS`
+? Choose authentication method used for APNs `Key`
```

Comment:
The user can also chose to use `Certificate` as the authentication method here, which uses a `.p12` file. 
I think we should also add its workflow here.